kde-misc/kdialog/kdialog-22.12.2.recipe

```diff
@@ -0,0 +1,118 @@
+SUMMARY="Display dialog boxes from shell scripts"
+DESCRIPTION="kdialog allows you to display dialog boxes from shell scripts.
+The syntax is very much inspired from the 'dialog' command (which shows text mode dialogs)."
+HOMEPAGE="https://invent.kde.org/utilities/kdialog/"
+COPYRIGHT="2006-2023 KDialog Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://invent.kde.org/utilities/kdialog/-/archive/v22.12.2/kdialog-v22.12.2.tar.gz"
+CHECKSUM_SHA256="35ad54955a48def6a13b726da17cf9b95119b794dd9ec183a403c197d25fc4de"
+SOURCE_DIR="kdialog-v$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	kdialog$secondaryArchSuffix = $portVersion
+	app:kdialog$secondaryArchSuffix = $portVersion
```

Comment:
```suggestion
	app:kdialog$commandSuffix = $portVersion
```

We need to verify that the command gets installed as `bin/kdialog` when building the `kdialog_x86` package on 32 bit Haiku.  You may have to adjust the cmake arguments in the BUILD() section or use a `mv` command in the INSTALL() section.

Reply:
I see it's already set up for use with `$commandBinDir` in the BUILD() section.  Would be nice to verify it after making these changes though.